tests/API1/testparamdepends.py

```diff
@@ -677,24 +677,23 @@ def test_param_external_param_instance(self):
         self.assertEqual(pinfo.name, 'a')
         self.assertEqual(pinfo.what, 'value')
     
-    @pytest.mark.skipif(sys.version_info.major == 2, reason='asyncio only on Python 3')
-    def test_async(self):
-        try:
-            param.parameterized.async_executor = async_executor
-            class P(param.Parameterized):
-                a = param.Parameter()
-                single_count = param.Integer()
-
-                @param.depends('a', watch=True)
-                @asyncio.coroutine
-                def single_parameter(self):
-                    self.single_count += 1
-
-            inst = P()
-            inst.a = 'test'
-            assert inst.single_count == 1
-        finally:
-            param.parameterized.async_executor = None
+    # @pytest.mark.skipif(sys.version_info.major == 2, reason='asyncio only on Python 3')
```

Comment:
@maximlt I think it might just be worth adding a comment to say this test can uncommented when python 2 is dropped.